infrastructure/iac/template/assets/icons.go

```diff
@@ -0,0 +1,34 @@
+package iac
+
+import (
+	"html/template"
+
+	"github.com/snyk/snyk-ls/domain/snyk"
+)
+
+func GetSeverityIconSvg(issue snyk.Issue) template.HTML {
```

Comment:
Wondering, if we should DRY the icons by moving them to `internal/html/severity_icons.go`, as we use the same severity icons everywhere.

Reply:
Always wanted to! I can do this!!! Thank youuuu!